src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java

```diff
@@ -25,6 +40,10 @@
     static final String currentUserDirectory = "CurrentUser";
     static final String myCertificateStore = "My";

```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.naming.ConstantNameCheck
**Severity**: ERROR
**File**: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java L41

Name &apos;myCertificateStore&apos; must match pattern &apos;^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$&apos;.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1302538922>*